cmd/soroban-cli/src/commands/network/mod.rs

```diff
@@ -86,8 +97,18 @@ impl Cmd {
             Cmd::Add(cmd) => cmd.run()?,
             Cmd::Rm(new) => new.run()?,
             Cmd::Ls(cmd) => cmd.run()?,
-            Cmd::Start(cmd) => cmd.run().await?,
-            Cmd::Stop(cmd) => cmd.run().await?,
+            Cmd::Container(cmd) => cmd.run().await?,
+
+            // TODO Remove this once `network start` is removed
+            Cmd::Start(cmd) => {
+                println!("⚠️ Warning: `network start` has been deprecated. Use `network container start` instead");
```

Comment:
I think this should go to stderr because it's not the main output of the command.
```suggestion
                eprintln!("⚠️ Warning: `network start` has been deprecated. Use `network container start` instead");
```